README.md

```diff
@@ -57,7 +57,7 @@ protocol.custom-init=39
 Now we need to set up LNDK. To start:
 
 ```
-git clone https://github.com/carlaKC/lndk
+https://github.com/lndk-org/lndk
```

Comment:
do we still want the git clone part?

Reply:
Yeah I think we can leave clone there until just before first release where we have release binaries and further instructions.

Building from source (via `cargo install lndk`) and cloning and building will always be an option though! :)

Reply:
oops 🤦‍♀️